java.jdbc-ballerina/build.gradle

```diff
@@ -247,6 +244,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push"
                 }
             }
+        } else {
+            throw new InvalidUserDataException("Central Access Token is Not Present")
```

Comment:
Shall we add the condition to check the `SNAPSHOT` in the version?
(Ref: https://github.com/ballerina-platform/module-ballerina-encoding/pull/80)

Reply:
It has been added in d8d4d89